main.py

```diff
@@ -385,6 +386,12 @@ async def get_and_delete_old_versions(image_name: str, inputs: Inputs, http_clie
                     # Skipping because this image version is tagged with a protected tag
                     delete_image = False
 
+            if inputs.dry_run:
+                delete_image = False
+                image_name_with_tag = f'{image_name}:{version.id}'
+                print(f'Would delete image {image_name_with_tag}.')
+                deleted.append(image_name_with_tag)
```

Comment:
Sorry, trying to review between other tasks, so just didn't have the time to look at whether there was a benefit. If there is, I'm open to keeping it :+1:

Reply:
Ah ok, I didn't manage to find the 2 seconds to pull it out last night either. The goal was to have the same data in place at the end of the run as what would have happened if the images really had been deleted.

That seems useful to me, but I can definitely see the reasoning to remove that line because the image wasn't really deleted and so to not have to maintain doing that in two places, though. I could also throw a quick comment before that line explaining why it's being done. Whatever your preference is, I'm happy to go with

Reply:
Since the list is used [here](https://github.com/snok/container-retention-policy/blob/main/main.py#L537:L545) to output which images were actually deleted, I suggest we remove it :+1: Could be misleading otherwise I think

Reply:
Done.